lms/djangoapps/discussion/views.py

```diff
@@ -365,6 +371,9 @@ def _find_thread(request, course, discussion_id, thread_id):
     if is_commentable_divided(course.id, discussion_id, course_discussion_settings) and not is_moderator:
         user_group_id = get_group_id_for_user(request.user, course_discussion_settings)
         if getattr(thread, "group_id", None) is not None and user_group_id != thread.group_id:
+            log.info("user_group:{user_group} is not equal to thread_group:{thread_group}".format(
```

Comment:
I would add a commong string in the log to identify these logs in splunk. 

e.g.
`Discussion error: user_group:{user_group} is not equal to thread_group:{thread_group}"`
`Discussion error: Thread Context:{context} for thread: {thread}`
`Discussion error: Thread with ID:{thread_id} not found for Discussion Module`